.github/labels.js

```diff
@@ -66,6 +66,31 @@ module.exports = [
         color: "912E29",
         description: ""
     },
+    {
+        name: "priority:blocker",
+        color: "cc0000",
+        description: "Blocks development and/or testing work, production could not run"
+    },
+    {
+        name: "priority:critical",
+        color: "ff0000",
+        description: "Crashes, loss of data, severe memory leak"
```

Comment:
Add security holes?

Reply:
Good point. We should add somewhere that users should not report security issues via Issues but report it via official ASF channels. Optionally we may enable GitHub feature for that

Reply:
We have it out of the box with GitHug issues - https://github.com/apache/maven-gh-actions-shared/issues/new/choose

<img width="1171" alt="image" src="https://github.com/user-attachments/assets/b677e07d-773c-493c-92b0-56cac3ebe484" />